### DISCUSSION TITLE
Multi-Select with values that have commas

### DISCUSSION THREAD

**Discussion by FNG-2002** [pocketbase/pocketbase#2197] [Q&A] [answered]
Hello,

is it possible to add a value to the multi select that contains commas?

For example: `["Test1", "Test2, Test2.1", "Test3", ...]`

I have values that need to be split (like"Test1" and "Test3") but I also have values, that kind of fit together that I want to seperate with a comma inside the multi select.

Is this possible or planned in the future?

Thanks in advance

**Comment by ganigeorgiev** [comment, marked as answer]
Technically it is possible, but I'm not sure if it is worth complicating the UI.

A select option is usually intended to represent 1 state/value and manually combining multiple values into 1 option kind of defeat the purpose of a multi-`select` in the first place. Doing so will complicate the filter conditions (you'll have to rely on the `~` operator) and you may also end up with partial duplicated selection, eg. `["test1", "test1,test2"]`.

So for now my recommendation would be to not manually combine values OR if you really need it - use a different delimiter like `-`, `|`, `/`, `;`, etc.

**Reply by FNG-2002** [reply]
Thanks for your input. I will consider using a different delimiter or split up the options.

**Reply by jimafisk** [reply]
Hi @ganigeorgiev,

I have a single select field where one of my options has a comma in it. I don't need it to represent multiple values, the comma just happens to be in the name.

I figured I would be able to either wrap the entire option in quotes or escape it with a backslash, but either way it still seems to split into separate options at the comma.

For example: Environment, 'Agriculture, Fishing and Forestry', Health

<details>
<summary>See screenshot</summary>

![comma](https://github.com/pocketbase/pocketbase/assets/5913244/7d7c8bc9-e73d-4dac-acee-a00299cfbb78)


</details>

Is there another way to do this? I wasn't sure how to use a custom delimiter. Thanks!

**Reply by ganigeorgiev** [reply]
@jimafisk At the moment there is no other option and custom delimiters are not supported as I really don't want to complicate the inline editing UI. I'll experiment later today to see whether an escape character (eg. `\`) could be used.

**Reply by ganigeorgiev** [reply]
An escape character (`\`) support was added in [v0.18.4](https://github.com/pocketbase/pocketbase/releases/tag/v0.18.4).

**Reply by jimafisk** [reply]
Worked like a charm! Thanks for the quick fix Gani!
